### ISSUE TITLE
Hacktoberfest

### ISSUE THREAD

**Issue by er888kh** [TheAlgorithms/Rust#376]
With Hacktoberfest coming, I thought it might be a good idea to devise a strategy beforehand so that our contributors and community can make the most out of this event.

There are many, many algorithms out there, with variable degree of utility, that can be implemented by contributors, but most of them won't be of particular value, even though they can't be rejected. For example we won't be benefiting from yet another sorting algorithm with terrible run time behavior that was mentioned in `$textbook`.

As such, it might be a good idea to somehow curate a to-do list with tasks of various difficulty levels for participants. The tasks need not be strictly coding related, but I can think of at least a few areas where we could benefit from having lots of new code (bignum, random and crypto to name a few.)

We can also start thinking of a new direction: Something like collecting all the accumulated knowledge here in a book or blog like platform to guide not only new learners, but also enthusiasts that could benefit from "human-readable" implementations of complex systems (like cryptography or random number generation for instance). One thing to keep in mind with this approach is that it will take a lot of polishing before it becomes anything useful.

To summarize, we need a strategy to make the most of this year's hacktoberfest. At very least a to-do list for new code and algorithms, but we may decide (and vote) to start being something more than a "just" repository to store algorithms. I think this issue might be a good place to start discussing our options.

**Comment by AnshulMalik** [comment]
True @er888kh, agree with you. We should plan ahead, have some issues created with varying difficulty.

I can think of a few things apart from the obvious algorithm implementations:
- Animations (maybe interactive https://raft.github.io/ is a good example) along with implementation 
- Real world example of simple algorithms being used to power rockets in space.

**Comment by siriak** [comment]
There's a website already https://the-algorithms.com/ with more verbose explanations. Animations and book-like descriptions would fit there, but not here next to source code. It'd be difficult to tell apart algorithms from their animations/complementary material in a single repo IMO. One thing that I'd like to point out is that adding something more than just algorithms would require a lot of effort and time, which I don't have. So any help would be greatly appreciated.
